runtime/vm/resolvesupport.cpp

```diff
@@ -2280,3 +2280,34 @@ resolveInvokeDynamic(J9VMThread *vmThread, J9ConstantPool *ramCP, UDATA callSite
 	Trc_VM_resolveInvokeDynamic_Exit(vmThread, result);
 	return result;
 }
+
+#if JAVA_SPEC_VERSION >= 16
+/**
+ * @brief The function calls into the interpreter via sendResolveUpcallInvokeHandle()
+ * to fetch the MemberName object plus appendix intended for the upcall method.
+ *
+ * @param vmThread the pointer to J9VMThread
+ * @param data the pointer to J9UpcallMetaData
+ * @return void
+ */
+void
+resolveUpcallInvokeHandle(J9VMThread *vmThread, J9UpcallMetaData *data)
+{
+	j9object_t invokeCache = NULL;
+	Trc_VM_resolveUpcallInvokeHandle_Entry(vmThread);
+
+	sendResolveUpcallInvokeHandle(vmThread, data);
+	invokeCache = (j9object_t)vmThread->returnValue;
+	if (NULL != vmThread->currentException) {
```

Comment:
Please use the VMHelpers helper.

Reply:
Replaced with `VM_VMHelpers::exceptionPending(vmThread)` as suggested above.